features/upgrade/etcd/upgrade.feature

```diff
@@ -0,0 +1,35 @@
+Feature: basic verification for upgrade testing
+  # @author geliu@redhat.com
+  # @case_id OCP-22606
```

Comment:
I think we want @case_id tag only before the check scenario. Otherwise our tools might be confused.

Reply:
ok

Reply:
I deleted @case_id tag for upgrade-prepare, and kept @author tag, I think this will help to trace the case author, 
and I also curious about @case_id tag have been masked by '#', it will be ignored by code, right?

Reply:
It is used by script to automatically update polarion, see DOC-1125811. Please use script instead of manually writing the YAML. Much easier and you make sure your tags are correct.